### ISSUE TITLE
RadioReference import doesn't show all county systems

### ISSUE THREAD

**Issue by tobfromme** [DSheirer/sdrtrunk#1635]
**Is your feature request related to a problem? Please describe.**
In RadioReference, there are county systems in the database that aren't shown within the RR import feature.  For example, "Local Schools" from this page are not shown in SDRTrunk:

https://www.radioreference.com/db/browse/ctid/2048#cid-5850

**Describe the solution you'd like**
The ability to select all systems within the RR database.

**Additional context**
RR post that details the issue:

https://forums.radioreference.com/threads/sdrtrunk-rr-conventional.462058/

**Comment by GTR8000** [comment]
I'll chime in here for clarification, since I posted in that thread.

SDRTrunk has a tab for importing what we call "county agency" pages in the RRDB, which are essentially sub-pages of the main county page. However what is not being shown anywhere are the listings on any main county page in the RRDB. The "Local Schools" that is being referenced is actually a subcategory of the main county page, not a separate agency page.

**tl;dr** there really ought to be a new tab in SDRTrunk showing all of the listings assigned directly to the county itself, not just the agency sub-pages
